src/Report/ReportSummary.php

```diff
@@ -106,7 +106,7 @@ public function getMemory()
     }
 
     /**
-     * @return time
+     * @return int
```

Comment:
Not really related, but I didn't want to make a PR out of this little tweak only.
I can revert it if you want.